src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -429,6 +430,19 @@ define(function (require, exports, module) {
         callback();
     }
 
+    // Based on http://stackoverflow.com/questions/21797299/convert-base64-string-to-arraybuffer
```

Comment:
Why not put this in [`FilerUtils.js`](https://github.com/timmoy/brackets/blob/7c744eb5675c7243d749c62a26c4cf07d9c80b41/src/filesystem/impls/filer/FilerUtils.js), since you're not using it in here at all?